src/Admin/AbstractAdmin.php

```diff
@@ -1731,11 +1724,6 @@ final public function getListMode(): string
         return $this->getRequest()->getSession()->get(sprintf('%s.list_mode', $this->getCode()), 'list');
     }
 
-    final public function getAccessMapping(): array
```

Comment:
You should mention this inside the CHANGELOG.

Reply:
Done